server/channels/app/channel.go

```diff
@@ -1829,17 +1829,26 @@ func (s *Server) getChannel(c request.CTX, channelID string) (*model.Channel, *m
 }
 
 func (a *App) GetChannels(c request.CTX, channelIDs []string) ([]*model.Channel, *model.AppError) {
-	channels, err := a.Srv().Store().Channel().GetMany(channelIDs, true)
-	if err != nil {
-		var nfErr *store.ErrNotFound
-		switch {
-		case errors.As(err, &nfErr):
-			return nil, model.NewAppError("GetChannel", "app.channel.get.existing.app_error", nil, "", http.StatusNotFound).Wrap(err)
-		default:
-			return nil, model.NewAppError("GetChannel", "app.channel.get.find.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
-		}
-	}
-	return channels, nil
+    channels, err := a.Srv().Store().Channel().GetMany(channelIDs, true)
+    if err != nil {
+        var nfErr *store.ErrNotFound
+        switch {
+        case errors.As(err, &nfErr):
+            return nil, model.NewAppError("GetChannel", "app.channel.get.existing.app_error", nil, "", http.StatusNotFound).Wrap(err)
+        default:
+            return nil, model.NewAppError("GetChannel", "app.channel.get.find.app_error", nil, "", http.StatusInternalServerError).Wrap(err)
+        }
+    }
+
+    // Filter out archived channels
+    activeChannels := []*model.Channel{}
```

Comment:
@TheInvincibleRalph This app method is used by other endpoints and functions in Mattermost and this would be a breaking change.  Have you looked at the redux selector that is used to display the channels on the frontend? You could filter it out there. I don't think the outgoing webhooks page makes any special call to fetch channels

Reply:
> @TheInvincibleRalph This app method is used by other endpoints and functions in Mattermost and this would be a breaking change. Have you looked at the redux selector that is used to display the channels on the frontend? You could filter it out there. I don't think the outgoing webhooks page makes any special call to fetch channels

I'll look into that and get back. Thank you for the review!